code/modules/projectiles/ammo_datums.dm

```diff
@@ -945,6 +945,36 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 /datum/ammo/bullet/shotgun/buckshot/on_hit_mob(mob/M,obj/projectile/P)
 	staggerstun(M, P, weaken = 2 SECONDS, stagger = 2 SECONDS, knockback = 2, slowdown = 0.5, max_range = 3)
 
+/datum/ammo/bullet/hefa_buckshot
+	name = "hefa fragment"
+	handful_icon_state = "shotgun buckshot shell"
+	icon_state = "buckshot"
+	hud_state = "shotgun_buckshot"
+	bonus_projectiles_type = /datum/ammo/bullet/hefa_buckshot/spread
+	bonus_projectiles_amount = 5
+	bonus_projectiles_scatter = 4
+	accuracy_var_low = 9
+	accuracy_var_high = 9
+	accurate_range = 3
+	max_range = 10
+	shrapnel_chance = 15
+	damage = 30
+	damage_falloff = 3
+
+/datum/ammo/bullet/hefa_buckshot/on_hit_mob(mob/mob_hit, obj/projectile/projectile)
+	staggerstun(mob_hit, projectile, knockback = 2, max_range = 4)
+
+/datum/ammo/bullet/hefa_buckshot/spread
+	name =	"hefa extra fragment"
```

Comment:
why does this use bonus projectiles?

Reply:
To give it an actual "spread" effect  that shrapnel has cause without it just ends up being a shit version of a slug but in 360.  Also the additional projectiles have the exact same stats as the base.